competency-explorer/src/components/MatrixView/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import cx from 'classnames';
+import { useSelector } from 'react-redux';
+import { selectDisplayMode } from '../../selectors/filters';
+import { MatrixLabel } from './MatrixLabel';
+import { selectVisibleCategories, selectVisibleLevels } from '../../selectors/visibility';
+import { MatrixCell } from './MatrixCell';
+import './styles.scss';
+
+export const MatrixView: React.FC = () => {
+    const curDisplayMode = useSelector(selectDisplayMode);
+    const categories = useSelector(selectVisibleCategories);
+    const levels = useSelector(selectVisibleLevels);
+     
+    const children = [<span className='empty' />];
```

Comment:
React is reporting the angry `unique "key" prop` complaint for this component. Maybe it's this `span`?

Reply:
thanks for flagging; I'll take a look in the next PR